# cancer-notes-and-code_review-and-read-ahead

weekend/extra reviewing/studying
papers, code, notes on sequencing, SVs, databases, pipelines

# Solid tumors
<< a table of some sorts here >>

# Papers
tags: `#immunesignatures` `#rnaseq`
- [ ] [2020 - RNA Immune Signatures from Pan-Cancer Analysis Are Prognostic for High-Grade Serous Ovarian Cancer and Other Female Cancers](https://www.mdpi.com/2072-6694/12/3/620) `#immunesignatures` `#rnaseq`

# Definitions
* __Subclonal__ "In cancer, subclones are __distinct subpopulations of cells within a tumor__ that have different genetic and/or epigenetic characteristics, originating from a common ancestral cell. These subclones can develop through the accumulation of mutations and other changes as cancer cells evolve and adapt to their environment. Understanding subclones is crucial because they can influence tumor behavior, treatment response, and resistance."

* __Clonal Hematopoiesis (CH)__ "Clonal hematopoiesis (CH) happens when a cell called a hematopoietic stem cell, which can develop into different types of blood cells, starts making cells with the same genetic mutation. These blood cells have a different genetic pattern than the rest of your blood cells." [source](https://www.mskcc.org/cancer-care/types/leukemias/risk-factors/clonal-hematopoiesis-ch)

<!-- words that come up that i mostly know to define/look into **soon** like over the weekends?? making a dictionary for myself for the future -->

* Canonical pathway
* Chipseq for validation when RNA-seq doesn’t give enough information
* Model system in a cell line — overexposes the function and mutant and see how it effects the growth
* Immune deconvolution
* Allelic expression — what is the other allele doing
* Tumor suppressor
* Gain of function
* LOH
* Truncating mutation
* Frameshift mutations
* Locus
* Functional effect, e.g. mutation does or does not have a functional effect
* MSI
* N-terminal path
* TMB - tumor mutational burden
* Knockout gene
* Wild type allele
* Clonal vs subclonal
* TCGA — associate with phenotype, additional cohorts
* Phenotypic difference
* Clinical phenotype
* Survival curve/survival analysis
* Oncogene vs. context dependent tumor suppressor activity
* How to interpret the data: mutually exclusive vs co-occuring patterns
* Validating expression
* Does the gene tolerate loss of function mutation
* Splice mutations/alteration
* Protein stabilization
* Misregulation
* Deleterious
* Gain of function
* Known databases - COMIC and oncoKB
* RNA-seq to see if alteration was expressed, possible to see no expression but still have an effect
* Ubiquinated
* In-frame-deletion
* misannotated mutation
* Exon skipping
* homdels
* Candidate testing
* Recurrent alteration
* For splice site mutations at MSKCC, the p.X_splice format follows this pattern: Format: p.[AminoAcid][Position]_splice
* Histology
* biallelic

# Etc
* [2008 - On the Process of Becoming a Great Scientist](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.0040033)

# Books (I'm Reading)
* `#BOC` - [The Biology of Cancer](https://wwnorton.com/books/9780393887655)
* `#GITC` - [Genomics in the Cloud](https://www.oreilly.com/library/view/genomics-in-the/9781491975183/)
* `#SEAG` - [Software Engineering at Google](https://www.oreilly.com/library/view/software-engineering-at/9781492082781/)
